app/helpers/sessions_helper.rb

```diff
@@ -4,15 +4,42 @@ def log_in user
   end
 
   def current_user
-    @current_user ||= User.find_by id: session[:user_id]
+    if user_id = session[:user_id]
+      @current_user ||= User.find_by(id: user_id) if session[:user_id]
```

Comment:
```suggestion
      @current_user ||= User.find_by(id: user_id) 
```
cần j code `if session[:user_id]` này em